core/kazoo_auth/priv/couchdb/providers/google.json

```diff
@@ -1,9 +1,30 @@
 {
     "_id": "google",
-    "enabled": true,
```

Comment:
for multi_factor I added this key, and in `kazoo_auth_mfa` it checks if the module is enabled by account or system.

Also the view `enabled_providers_by_type` is using this key to list enabled provider.